openmls/src/group/core_group/staged_commit.rs

```diff
@@ -5,7 +5,7 @@ use crate::framing::mls_content::FramedContentBody;
 use crate::treesync::errors::TreeSyncAddLeaf;
 use crate::treesync::node::encryption_keys::EncryptionKeyPair;
 use crate::treesync::node::leaf_node::{
-    LeafNodeTbs, OpenMlsLeafNode, TreeInfoTbs, VerifiableLeafNodeTbs,
+    LeafNodeTbs, OpenMlsLeafNode, TreeInfo, VerifiableLeafNodeTbs,
```

Comment:
This is an unnecessary change and unrelated to moving code around.

Reply:
Can we keep this change? The `TreeInfo` is not signed and this PR is about cleaning up.

Reply:
We discussed this offline and I'll revert this commit for now.